# Just a example yaml config file
# You should put this configutation at ~/husky_navigation/config/planner.yaml

RRTStarPlanner:
  goal_tolerance: 0.2
  radius: 1.05
  epsilon: 0.21
  max_num_nodes: 50001
  min_num_nodes: 5501
